packages/themes/src/tokens.js

```diff
@@ -5,86 +5,184 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+const _colors = {
```

Comment:
@joshblack ready for review / feedback:

Sync task output for "api documentation": https://github.com/IBM/carbon-elements/tree/f6b24e668ef4d38c7d55a1be9064bfad967375c5/packages/elements#-api-documentation

I added docs to elements package. Colors and themes Sassdoc is done. Anything in "general" group here are from packages that I haven't looked at yet: https://github.com/IBM/carbon-elements/blob/f6b24e668ef4d38c7d55a1be9064bfad967375c5/packages/elements/docs/sass.md

You can also see Sassdoc in each package, e.g.: https://github.com/IBM/carbon-elements/blob/f6b24e668ef4d38c7d55a1be9064bfad967375c5/packages/themes/docs/sass.md

The main thing I want reviewed is the new `_colors` object in themes/src/tokens.js. It's used to generate the current `colors` export so there are no breaking changes. It's also used for a new `tokenDocs` export. `_colors` is meant to be the source for token metadata used in Sassdoc and eventually this website content: https://www.carbondesignsystem.com/guidelines/color/usage

Side note, I think there are opportunities to better author and document `@carbon/colors`... but the goal if this PR is to introduce as few new public items as possible

Reply:
One thing I want to be careful about with the JS files is to set them up so that folks can use them in prod if they want to. We should try and find a good strategy for how to deliver meta information without coupling it too much to what we want people to ship to users.

Are there any good conventions that we could follow? Maybe a separate `meta.js` file setup? Haven't thought too much about this so whatever you think would work best totally works 👍

We should copy whatever this strategy is over in our icon repo. Currently, we have a `metadata.yml` file over in that project to allow authorship to be a little easier for design to contribute.

Reply:
For now, I moved data out of `tokens.js` and into a `metadata.yml` that `tasks/build.js` consumes. Since it's no longer in exported JS... I wouldn't expect product teams to use the meta data... so we have the flexibility to modify meta data approach in the future along with icons, etc. if necessary.